sequence_processing_pipeline/Pipeline.py

```diff
@@ -207,13 +209,17 @@ def validate(self, sample_sheet_path):
             for item in val_sheet.samples:
                 unique_index = f'{item.lane}_{item.sample_id}'
                 if unique_index in unique_indexes:
+                    passes_additional_tests = False
                     msgs.append(ErrorMessage("A sample already exists with la"
                                              f"ne {item.lane} and sample-id "
                                              f"{item.sample_id}"))
```

Comment:
I think it would be easier to do here `val_sheet = None` and just do `return msgs, val_sheet` without the if.

Reply:
The only problem is we're iterating through val_sheet.samples while we check, so it would lead to an error after the first error is encountered.

Reply:
add a brake to the for loop?

Reply:
but then we'd only catch the first error, which is different behavior than all other cases where we return a list of all errors and warnings.

Reply:
Got it, for whatever reason I thought ErrorMessage will raise an exception/error but it doesn't. Thank you for the explanation